dash/testing/browser.py

```diff
@@ -39,7 +44,7 @@ def __init__(
         self._download_path = download_path
         self._wait_timeout = wait_timeout
 
-        self._driver = self.get_webdriver(remote)
+        self._driver = until(lambda: self.get_webdriver(remote), timeout=1)
```

Comment:
@alexcjohnson   I change the hardcode 3x trial with the `wait.until` mechanism, I can try later on remove this and get more info from the logs for debugging

Reply:
Oh I see, `get_webdriver` swallows the error. I suppose that's OK, but the other way has the advantage that if the error persists all the way to the end it's still raised as a regular exception. perhaps build this in (optionally) to `until`, or make a new `until` variant `until_no_error`?